src/index.js

```diff
@@ -362,7 +362,7 @@ function parseGraphQLParams(
 
   // Parse the variables if needed.
   let variables = urlData.variables || bodyData.variables;
-  if (typeof variables === 'string') {
+  if (variables !== null && variables !== '' && typeof variables === 'string') {
```

Comment:
thanks @asiandrummer - incorporated and tested the changes. looks good so far.

 if (variables && typeof variables === 'string') {
   ...
 }